### DISCUSSION TITLE
Changing Logging Type for One Type of Resource

### DISCUSSION THREAD

**Discussion by reza8iucs** [Azure/Enterprise-Scale#1226] [Q&A]
[](url)There is currently an ALZ policy that forces resources to send their diagnostic logs to Log Analytics in the management subscription. That's great but how do i change this behavior for an specific type of resource say Logic Apps?. Here is my requirement:

Logic Apps team recently announced the availability for sending Workflow Runtime logs to Log Analytics. Default setbypolicy only send the Function Application Logs to Log Analytics, I need to change it in away that Workflow Runtime Logs are sent to Log Analytics instead, as shown in the picture attached.

Thanks for your help

![image](https://user-images.githubusercontent.com/110741554/220350780-e523a557-51ee-43e5-b57e-637de23f38c0.png)

**Comment by jtracey93** [comment]
We will need to add the category in a PR to the policy (https://github.com/Azure/Enterprise-Scale/blob/main/src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-Function.json) i believe. 

Please create an issue and let us know the policy that actually sets these settings so we can pin point the correct one :)

**Reply by reza8iucs** [reply]
Thanks Jack...I will make the changes in my repo and if it works I can create a pull request here. 

What's the correct direction? Do i change this in my GitHub repo and push or should i change it in the portal policy?

**Reply by jtracey93** [reply]
Perfect. 

You can change it in the portal for the quickest change and testing approach
